src/app/dataset-block/metadata-block/components/event-details/components/common/yaml-event-viewer/yaml-event-viewer.component.ts

```diff
@@ -1,8 +1,10 @@
+import { map } from "rxjs/operators";
 /* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
Please get rid of the pragma. Don't tell it's impossible.

Reply:
It's possible, fixed